### ISSUE TITLE
Uikit v3 admin theme color contrast of admin notes does not meet minimum color contract ratio suggested by WCAG

### ISSUE THREAD

**Issue by heybran** [processwire/processwire-issues#1928]
### Short description of the issue

`Uikit v3 admin theme` (version: `0.3.3`) admin notes text color and background color does not have enough color contract.

### Expected behavior

Based on [Color contrast](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast), Minimum color contract ratio (AA rating) of body text should be `4.5 : 1`

### Actual behavior

Existing color contract ratio on notes is `3.0.3`

### Optional: Screenshots/Links that demonstrate the issue

<img width="1433" alt="image" src="https://github.com/processwire/processwire-issues/assets/75633537/8c7f0c22-fa44-4090-a59d-d43049b9a829">

### Optional: Suggestion for a possible fix

Update `#main .notes` color to be `#70757d` to increase the contract ratio.

<img width="979" alt="image" src="https://github.com/processwire/processwire-issues/assets/75633537/a7e3ed4f-4b8f-49ff-b6c4-f52131c0f911">

### Steps to reproduce the issue

1. Log into admin
2. Check yellow notes as described above
3. Use FireFox dev tool or other tools to check color contract against WCAG standards.

### Setup/Environment

* ProcessWire version: 3.0.229
* (Optional) PHP version: 
* (Optional) MySQL version: 
* (Optional) Any 3rd party modules that are installed and could be related to the issue:

**Comment by heybran** [comment]
Sorry, closing this for now, as I found texts on other areas also have contract issue, so it doesn't make sense to only mention this one at the moment. I will help report other texts contract once I play more with ProcessWire and then update this issue.

**Comment by BernhardBaumrock** [comment]
Hey @heybran we have tried to improve accessibility in AdminStyleRock, so if you find time you might try out the dev branch of AdminStyleRock and let us know if you find any room for improvement! https://github.com/baumrock/AdminStyleRock/tree/dev

**Comment by heybran** [comment]
Hey @BernhardBaumrock sure thing, I will check it out.
